packages/api/src/services/create_group.ts

```diff
@@ -91,3 +91,59 @@ export const getRecommendationGroups = async (
 export const getInviteUrl = (invite: Invite) => {
   return `${homePageURL()}/invite/${invite.code}`
 }
+
+export const joinGroup = async (
```

Comment:
Maybe should be in a file name `join_group` or we could rename this file to `groups`

Reply:
Renamed this file to `groups.ts` @jacksonh